embed/config.go

```diff
@@ -290,8 +290,8 @@ type Config struct {
 	ListenMetricsUrls     []url.URL
 	ListenMetricsUrlsJSON string `json:"listen-metrics-urls"`
 
-	// Logger is logger options: "zap", "capnslog".
-	// WARN: "capnslog" is being deprecated in v3.5.
+	// Logger is logger options: currently only supports "zap".
```

Comment:
@jingyih awesome! small comment here and below - since we are now removing it, the word deprecated sounds confusing (which could be understood as to be removed in future).

Reply:
Thanks! Your suggestion is applied.